Description: Fix some problems in build.xml and link against system installed
 javadocs.
Forwarded: not-needed
Author: Onkar Shinde <onshinde@ubuntu.com>
--- a/build.xml
+++ b/build.xml
@@ -64,7 +64,7 @@
       </fileset>
     </copy>
     <copy todir="${classesdir}">
-      <fileset dir="home/niclas/dev/opensource">
+      <fileset dir=".">
         <include name="LICENSE.txt">
         </include>
       </fileset>
@@ -105,12 +105,12 @@
         </include>
       </fileset>
     </copy>
-    <copy todir="${testclassesdir}">
+    <!--copy todir="${testclassesdir}">
       <fileset dir="src/test/conf">
         <include name="**/*">
         </include>
       </fileset>
-    </copy>
+    </copy-->
   </target>
   <target name="jar" description="o Create the jar" depends="compile,test">
     <jar jarfile="target/${final.name}.jar" excludes="**/package.html" basedir="${classesdir}">
@@ -204,6 +204,10 @@
         <pathelement location="target/${final.name}.jar">
         </pathelement>
       </classpath>
+      <link offline="true" packagelistLoc="/usr/share/doc/default-jdk-doc/api/" href="/usr/share/doc/default-jdk-doc/api/" />
+      <link offline="true" packagelistLoc="/usr/share/doc/libmail-java/api/" href="/usr/share/doc/libmail-java/api/" />
+      <link offline="true" packagelistLoc="/usr/share/doc/liblog4j1.2-java/api/" href="/usr/share/doc/liblog4j1.2-java/api/" />
+      <link offline="true" packagelistLoc="/usr/share/doc/libservlet3.1-java/api/" href="/usr/share/doc/libservlet3.1-java/api/" />
     </javadoc>
   </target>
   <target name="get-deps" unless="noget" depends="init">
@@ -228,4 +232,4 @@
     <unjar dest="${maven.home}" src="${user.home}/maven-install-latest.jar">
     </unjar>
   </target>
-</project>
\ No newline at end of file
+</project>
